scripts/install.js

```diff
@@ -30,10 +30,29 @@ var recreateDir = function (dirPath) {
   cleanDir(dirPath);
   createDir(dirPath);
 };
-
-var prepareFiles = function () {
+var OFFLINE_PACKAGE_SUFFIX = "offline";
+var prepareOffLinePackageJSON = function () {
+  var offlinePackageJSON = require("../offline-package.json");
+  var packageJSON = require("../package.json");
+  offlinePackageJSON.version = packageJSON.version;
+  fs.writeFileSync(path.resolve("./offline-package.json"), JSON.stringify(offlinePackageJSON, null, 2), "utf-8" );
+  fs.copyFileSync(path.resolve("./package.json"), path.resolve("./package-backup.json"));
+  fs.copyFileSync(path.resolve("./offline-package.json"), path.resolve("./package.json"));
+};
+var prepareFiles = function (buildOffLinePakcage) {
   var buildDir = localPath("build"),
     copyList = ["gauge-proto", "src", "skel", "index.js", "index.bat", "debug.bat", "js.json", "package.json", "package-lock.json", ".node-inspectorrc", "README.md"];
+  if( buildOffLinePakcage) {
+    prepareOffLinePackageJSON();
+    try {
+      console.log("Installing dependencies...");
+      child_process.execSync("npm install", { cwd: localPath() });
```

Comment:
We don't need to install the dev dependencies.

Reply:
Fixed